transfer/dummy.py

```diff
@@ -25,26 +26,30 @@ def main():
 
     train_data = train_data * fac + 0.01
     test_data = test_data * fac + 0.01
+
     train_labels = tf.keras.utils.to_categorical(train_labels, num_classes=NUMBER_OF_LABELS)
     test_labels = tf.keras.utils.to_categorical(test_labels, num_classes=NUMBER_OF_LABELS)
 
     model = tf.keras.models.Sequential()
-    model.add(tf.keras.layers.Conv2D(filters=2, kernel_size=(3, 3), activation='relu', input_shape=input_shape))
-    model.add(tf.keras.layers.MaxPooling2D())
-    model.add(tf.keras.layers.Conv2D(filters=2, kernel_size=(3, 3), activation='relu'))
-    model.add(tf.keras.layers.Flatten())
-    model.add(tf.keras.layers.Dense(activation='relu', units=50))
-    model.add(tf.keras.layers.Dense(activation='relu', units=30))
-    model.add(tf.keras.layers.Dense(activation='softmax', units=10))
-
-    model.compile(loss=tf.keras.losses.categorical_crossentropy,
-                  optimizer=tf.keras.optimizers.Adadelta(), metrics=['accuracy'])
-    history = model.fit(x=train_data, y=train_labels, batch_size=10, epochs=8, verbose=1)
-
-    result = model.predict(test_data[0:10])
-    result = [np.argmax(r) for r in result]
-    print("predictions: " + str(result))
-    print("actual: " + str([np.argmax(l) for l in test_labels[0:10]]))
+
+    model.add(tf.keras.models.Conv2D(32, kernel_size=(3, 3), activation='relu', input_shape=input_shape))
+
+    model.add(tf.keras.models.Conv2D(64, (3, 3), activation='relu'))
+
+    # reduce dim, choose best
+    model.add(tf.keras.models.MaxPooling2D(pool_size=(2, 2)))
+
+    #model.add(Dropout(0.25))
+    model.add(tf.keras.models.Flatten())
+
+    model.add(tf.keras.models.Dense(128, activation='relu'))
+
+    #model.add(Dropout(0.5))
+    model.add(tf.keras.models.Dense(NUMBER_OF_LABELS, activation='softmax'))
+    model.compile(loss=tf.keras.losses.categorical_crossentropy, optimizer=tf.keras.optimizers.SGD, metrics=['accuracy'])
+
+    history = model.fit(x=train_data, y=train_labels, batch_size=128, epochs=10, verbose=1,
+                        validation_data=(test_data, test_labels))
```

Comment:
the test_data should not be here. you can use validation_split = 0.2 or 0.1 instead